### ISSUE TITLE
Feature Request: Warnings shall be yellow

### ISSUE THREAD

**Issue by sustefil** [MISP/MISP#8240]
### Is your feature request related to a problem? Please describe.

Hi there, noticed the new Warning feature in the MISP events.

https://user-images.githubusercontent.com/3668672/159040894-99d951e3-a076-40c7-9bbd-9ff619df2e5c.png

The RED color is a hit in the eyes..

I am pretty used to see the RED color when the event is unpublished, this adds confusion for me

### Describe the solution you'd like

How about yellow? it's just a warning anyway :) 

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by iglocska** [comment]
Yeah, good point. It may have been "slightly" over the top, but the person that implemented it is an utter monkey when it comes to UX design. We'll change it to something more pleasant.
